Source/JavaScriptCore/debugger/Debugger.h

```diff
@@ -345,6 +331,7 @@ class Debugger : public DoublyLinkedListNode<Debugger> {
     JSValue m_currentException;
     CallFrame* m_pauseOnCallFrame { nullptr };
     CallFrame* m_currentCallFrame { nullptr };
+    JSGenerator* m_pauseInGenerator { nullptr };
```

Comment:
How is this object kept alive from GC?

Reply:
fundamentally all we're using this for is as a pointer identifier (i.e. we never dereference it)

generally speaking we also really don't like it when Web Inspector keeps things alive as that can make it much more difficult for developers to identify memory issues

would wrapping it in `Weak` work?